.github/actions/testing_report_generation/app.rb

```diff
@@ -135,5 +135,5 @@ def get_workflows(client, repo_name)
   client.add_comment(REPO_NAME_WITH_OWNER, last_issue.number,failure_report.get_report)
 # Create a new issue if there exists failed workflows.
 else
```

Comment:
Shouldn't this else clause be deleted?  Won't the `create_issue` at line 127 cover all cases where a new issue needs t be created?

Reply:
> @granluo See the question above

The comment was pending and has been updated.

Reply:
Thanks for the clarification.